src/components/BotSetting/BotLimit.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  TextFieldInput,
+  TextFieldRoot,
+  Typography,
+} from '@mochi-ui/core'
+import { UsageLimitTypeEnum } from '~/model/usage-limit-type'
+import { type BotSettingData } from './BotSetting'
+import { Controller, useFormContext } from 'react-hook-form'
+
+interface LimitOptions {
+  label: string
+  value: UsageLimitTypeEnum
+}
+
+const options: LimitOptions[] = [
+  { label: 'Per One Hour', value: UsageLimitTypeEnum.PerOneHour },
+  { label: 'Per Four Hours', value: UsageLimitTypeEnum.PerFourHours },
+  { label: 'Per Day', value: UsageLimitTypeEnum.PerDay },
+  { label: 'Per Month', value: UsageLimitTypeEnum.PerMonth },
+  { label: 'Per Week', value: UsageLimitTypeEnum.PerWeek },
+]
+
+export const BotLimit = () => {
+  const { control } = useFormContext<BotSettingData>()
+
+  return (
+    <div className="space-y-4">
+      <div className="flex flex-row items-end space-x-4">
+        <Controller
+          name="usageLimitPerUser"
+          control={control}
+          render={({ field, fieldState }) => (
+            <FormControl error={!!fieldState.error} hideHelperTextOnError>
+              <FormLabel>Usage limit per user</FormLabel>
+              <TextFieldRoot>
+                <TextFieldInput {...field} />
+              </TextFieldRoot>
+              <FormErrorMessage>{fieldState.error?.message}</FormErrorMessage>
```

Comment:
This looks off
![image](https://github.com/dwarvesf/chatbot-builder/assets/12707960/2c2b9c3d-62b2-47fc-b8f1-5836cc547edc)